shipyard_airflow/dags/drydock.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+### DryDock Dag
+"""
+import airflow
+from airflow import DAG
+from airflow.operators.sensors import HttpSensor
+from airflow.operators.bash_operator import BashOperator
+from datetime import timedelta
+import json
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': airflow.utils.dates.days_ago(2),
+    'retries': 1,
+    'retry_delay': timedelta(minutes=1),
+    'provide_context': True
+}
+
+dag = DAG('drydock_dag', default_args=default_args, schedule_interval="@once")
+
+# Location of site_yaml and prom_yaml
+site_yaml = '/home/ubuntu/demo-bundle/drydock.yaml'
+prom_yaml = '/home/ubuntu/demo-bundle/configs/complete-bundle.yaml'
+
+# Drydock Endpoint
+drydock_api_endpoint = 'http://drydock-api.drydock:9000'
+drydock_design_api = 'http://drydock-api.drydock:9000/api/v1.0/designs'
+drydock_task_api = 'http://drydock-api.drydock:9000/api/v1.0/tasks'
+
+# Drydock Token
+drydock_token = 'bigboss'
+
+
+# Add drydock connection to airflow
+add_connection = BashOperator(
+    task_id='add_drydock_connection',
+    bash_command='airflow connections --add --conn_id drydock_api --conn_uri {{ params.drydock_endpoint }}',
+    params={'drydock_endpoint': drydock_api_endpoint},
+    dag=dag)
+
+# Check to make sure that the drydock endpoint is reachable
+sensor = HttpSensor(
+    task_id='drydock_sensor_check',
+    http_conn_id='drydock_api',
+    endpoint='',
+    poke_interval=5,
+    dag=dag)
+
+# Get Design ID
```

Comment:
I understand this is a WIP and is likely testing the waters -- this seems like a clear case where we want to leverage a drydock python client instead of bash calls.